python/paddle/fluid/layers/nn.py

```diff
@@ -9083,3 +9084,50 @@ def get_tensor_from_selected_rows(x, name=None):
         outputs={'Out': out},
         attrs={})
     return out
+
+
+def psroi_pool(input, rois, output_channels, spatial_scale, pooled_height,
+               pooled_width):
```

Comment:
add name at last, better to make API consistent.

```
pooled_width, name=None):
```

Reply:
done